CHANGELOG.md

```diff
@@ -8,13 +8,18 @@ have notable changes.
 
 Changes since v2.12
 
+### Breaking changes
+- Multiple organization support for users #2035
+- Organizations are maintained in the database and not config #2039
```

Comment:
should we add migration instructions here? something like:
1. upgrade rems
2. add all the organizations form your config via the API
3. remove organizations from your config

what about creating the default organization for users who don't need the organizations feature?

Reply:
I was originally thinking to have a CLI tool but since we are about to build the organization UI too we could just point to that?

Reply:
fine by me!